ports/esp32/Makefile

```diff
@@ -791,6 +525,56 @@ vpath %.cpp . $(TOP)
 $(BUILD)/%.o: %.cpp
 	$(call compile_cxx)
 
+################################################################################
+# Port-specific functions table for dynamic linking
```

Comment:
Since this is not part of the main build, would be good to have it as a separate, included file like `esp32_port_fun.mk`